Modules/Sources/Yosemite/Stores/OrderCardPresentPaymentEligibilityStore.swift

```diff
@@ -83,10 +111,42 @@ private extension OrderCardPresentPaymentEligibilityStore {
     }
 }
 
+/// Error logging
+private extension OrderCardPresentPaymentEligibilityStore {
+    func logFailedStorageSiteRead(siteID: Int64, currentSiteFallbackValue: Site?) {
+        let message = "OrderCardPresentPaymentEligibilityStore: Storage site missing, falling back to currentSite."
+
+        DDLogError(message)
+
+        crashLogger.logMessage(
+            message,
+            properties: [
+                "siteID": siteID,
+                "currentSiteID": currentSiteFallbackValue?.siteID ?? "empty",
+            ],
+            level: .error
+        )
+    }
+
+    func logFailedDefaultSiteRead(siteID: Int64) {
+        let message = "OrderCardPresentPaymentEligibilityStore: Current default site missing."
+
+        DDLogError(message)
+
+        crashLogger.logMessage(
+            "OrderCardPresentPaymentEligibilityStore: Current default site missing.",
+            properties: [
+                "requestedSiteID": siteID
+            ],
+            level: .error
+        )
+    }
+}
+
 extension OrderCardPresentPaymentEligibilityStore {
     enum OrderIsEligibleForCardPresentPaymentError: Error {
         case orderNotFoundInStorage
-        case siteNotFoundInStorage
+        case failedToObtainSite
```

Comment:
Perhaps be slightly more descriptive here?

Not essential but may help future debugging/support

```suggestion
        case failedToObtainSiteForCIABFeatureCheck
```

Reply:
@joshheald Addressed in [8ebbd2e](https://github.com/woocommerce/woocommerce-ios/pull/16413/commits/8ebbd2ebeeca1630fa4d41798e713ae534a3d9a2). The flow now proceeds to orders check in case if a site wasn't obtained.